libs/core/langchain_core/tools.py

```diff
@@ -403,6 +380,16 @@ class ChildTool(BaseTool):
     two-tuple corresponding to the (content, artifact) of a ToolMessage.
     """
 
+    def __init__(self, **kwargs: Any) -> None:
+        """Initialize the tool."""
+        if "args_schema" in kwargs and kwargs["args_schema"] is not None:
+            if not is_basemodel_subclass(kwargs["args_schema"]):
+                raise SchemaAnnotationError(
```

Comment:
is this a SchemaAnnotationError, sounds like a type error or value error?

Reply:
do we need to update line 311 as well?

Reply:
Probably not in this PR for line 311 I don't know what its role was. Users could be relying on the identity of the exception